addons/webpack.buildAddons.js

```diff
@@ -4,9 +4,10 @@ var path = require('path')
 
 module.exports = {
   entry: {
-    createAutoCorrectedDatePipe: path.join(__dirname, './src/createAutoCorrectedDatePipe.js'),
-    createNumberMask: path.join(__dirname, './src/createNumberMask.js'),
-    emailMask: path.join(__dirname, './src/emailMask.js')
+    textMaskAddons: path.join(__dirname, './src/index.js'),
+    createAutoCorrectedDatePipe: [path.join(__dirname, './src/createAutoCorrectedDatePipe.js')],
```

Comment:
I know, this is an strange workaround for a limitation in Webpack 1 it seems... in Webpack 2 this is no longer the case... https://github.com/webpack/webpack/issues/300